src/Libraries/Microsoft.Extensions.Diagnostics.ResourceMonitoring/ITcpTableInfo.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Microsoft.Extensions.Diagnostics.ResourceMonitoring;
+
+/// <summary>
+/// An interface for getting TCP/IP table information.
+/// </summary>
+internal interface ITcpTableInfo
```

Comment:
I find the interface name somewhat confusing, and I think there's a room for improvement.

1. The interface doesn't have a verb, which somewhat implies it describes a DTO of some kind; yet it describes behaviours. A better name would be something like `ITcpTableInfoProvider`.
2. Inconsistent language - the interface name features "TcpTable", the methods' return types feature "TcpState", and the methods' names use "IP snapshots".

Reply:
Done